src/main/cpp/chassis/SwerveChassis.cpp

```diff
@@ -424,6 +450,51 @@ void SwerveChassis::LogInformation()
     Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, m_networkTableName, string("current rotation position"), pose.Rotation().Degrees().to<double>());
 }
 
+void SwerveChassis::DataLog()
+{
+
+    LogPoseData(DragonDataLoggerSignals::PoseSingals::CURRENT_CHASSIS_POSE, GetPose());
+
+    LogDoubleData(DragonDataLoggerSignals::DoubleSignals::CHASSIS_STORED_HEADING_DEGREES, GetStoredHeading().value());
+
+    frc::ChassisSpeeds targetSpeed;
+    targetSpeed.vx = m_drive;
+    targetSpeed.vy = m_steer;
+    targetSpeed.omega = m_rotate;
+    LogChassisSpeedsData(DragonDataLoggerSignals::ChassisSpeedSignals::TARGET_SPEEDS, targetSpeed);
+
+    auto currFrontLeftState = m_frontLeft->GetState();
+    auto currFrontRightState = m_frontRight->GetState();
+    auto currBackLeftState = m_backLeft->GetState();
+    auto currBackRightState = m_backRight->GetState();
+    wpi::array<frc::SwerveModuleState, 4> states = {currFrontLeftState, currFrontRightState, currBackLeftState, currBackRightState};
+    auto currentSpeed = m_kinematics.ToChassisSpeeds(states);
+    LogChassisSpeedsData(DragonDataLoggerSignals::ChassisSpeedSignals::ACTUAL_SPEEDS, currentSpeed);
+
+    auto optFrontLeftState = m_frontLeft->GetOptimizedState();
+    auto optFrontRightState = m_frontRight->GetOptimizedState();
+    auto optBackLeftState = m_backLeft->GetOptimizedState();
+    auto optBackRightState = m_backRight->GetOptimizedState();
+    LogSwerveModuleStateData(DragonDataLoggerSignals::SwerveStateSingals::TARGET_LEFT_FRONT_STATE, optFrontLeftState);
+    LogSwerveModuleStateData(DragonDataLoggerSignals::SwerveStateSingals::TARGET_LEFT_BACK_STATE, optBackLeftState);
+    LogSwerveModuleStateData(DragonDataLoggerSignals::SwerveStateSingals::TARGET_RIGHT_FRONT_STATE, optFrontRightState);
+    LogSwerveModuleStateData(DragonDataLoggerSignals::SwerveStateSingals::TARGET_RIGHT_BACK_STATE, optBackRightState);
+
+    LogSwerveModuleStateData(DragonDataLoggerSignals::SwerveStateSingals::ACTUAL_LEFT_FRONT_STATE, currFrontLeftState);
+    LogSwerveModuleStateData(DragonDataLoggerSignals::SwerveStateSingals::ACTUAL_LEFT_BACK_STATE, currBackLeftState);
```

Comment:
Actual Left Back state isn't being logged for some reason, can't find the error in the code

Reply:
Is that before or after my last fix?